### PR TITLE
ACP-21: Entry point activity and login screen

### PR DESCRIPTION
**Pull request by jonholmberg**

### CODE REVIEW

File: Customers/app/src/main/java/com/e_conomic/customers/ui/activities/CustomersBaseActivity.kt [hunk 1]
```diff
@@ -37,6 +40,18 @@ abstract class CustomersBaseActivity : AppCompatActivity() {
         setupUI()
     }
 
+    override fun onResume() {
+        super.onResume()
+        val session = applicationComponent.provideSession()
+        if (!session.isLoggedIn) {
```
Comment (jonholmberg) [inline Customers/app/src/main/java/com/e_conomic/customers/ui/activities/CustomersBaseActivity.kt]: Is this the correct check here?

File: Customers/app/src/main/java/com/e_conomic/customers/extensions/OkHttpClientExtensions.kt [hunk 2]
```diff
@@ -9,20 +10,18 @@ import java.io.IOException
...
-private val appId: String = "ihUXICNLYCmAJz_GJ1RaoGn3mA69XLzHJZeOh4zQ8Jo1"
-private val accessId: String = "8rQKKZFTmcOxWORvj4M6m3jHxacBMBDBOSBtBZ9Y7M01"
 private val contentType: String = "application/json"
 
-fun Request.Builder.addEconHeaders(): Request.Builder =
-        this.header("appId", appId)
-            .header("accessId", accessId)
-            .header("Content-Type", contentType)
+fun Request.Builder.addEconHeaders(headers: Map<String, String>): Request.Builder {
+    headers.forEach { key, value -> this.header(key, value) }
```
Comment (ned1988) [inline Customers/app/src/main/java/com/e_conomic/customers/extensions/OkHttpClientExtensions.kt]: Nice 👍

File: Customers/app/src/main/java/com/e_conomic/customers/ui/activities/CustomersBaseActivity.kt (hunk 1 above)
Comment (ned1988) [inline Customers/app/src/main/java/com/e_conomic/customers/ui/activities/CustomersBaseActivity.kt]: Looks fine for me, but maybe I can't see something.

Comment (jonholmberg) [inline Customers/app/src/main/java/com/e_conomic/customers/ui/activities/CustomersBaseActivity.kt]: Cool! :) I was just unsure whether this catches all the cases where the token is expired, account removed etc..

Comment (tudorgk) [inline Customers/app/src/main/java/com/e_conomic/customers/ui/activities/CustomersBaseActivity.kt]: In e-conomic the token **NEVER** expires :)

Comment (jonholmberg) [inline Customers/app/src/main/java/com/e_conomic/customers/ui/activities/CustomersBaseActivity.kt]: Ah, that's good to know!

**Comment by jonholmberg** [comment]
Anything else here @tudorgk?

**Comment by tudorgk** [comment]
Looks good

**Pull request merged**